release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1055,6 +1059,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* In the {rh-openstack-first}, a potential issue in Kuryr ports pools can occur when a variety of bulk port creation requests are made at the same time. During these bulk requests, if the IP allocation for one of the IPs fails, the operation will be retried for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pool batch settings to avoid a variety of bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
```

Comment:
```suggestion
* In the {rh-openstack-first}, a potential issue in Kuryr ports pools can occur when a variety of bulk port creation requests are made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the operation will be retried for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pool batch settings to avoid a variety of bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
```